test/JDBC/expected/datetime2fromparts-vu-prepare.out

```diff
@@ -0,0 +1,26 @@
+-- [BABEL-1627, 2827] Support DATETIME2FROMPARTS Transact-SQL function
+CREATE PROCEDURE datetime2fromparts_vu_prepare_p1 AS (select DATETIME2FROMPARTS(2011, 8, 15, 14, 23, 44, 5, 1 ));
```

Comment:
testing seems insufficient. Are we covering various boundary conditions, negatives and somewhere else?